application/plugins/sms_to_xmpp/libraries/abhinavsingh-JAXL-5829c3b/app/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>403 Forbidden</title>
+</head>
+<body>
+
+<p>Directory access is forbidden.</p>
+
```

Comment:
Do we really need these files here inside external libraries? 
Instead can't we just block access to the libraries folder?

Reply:
I guess it is a security measure in case server is not well configured. All folders, even in the system/ folder contain these files. So better keep them.

Reply:
"CodeIgniter will have an index.html file in all of its directories in an attempt to hide some of this data, but have it in mind that this is not enough to prevent a serious attacker."
https://codeigniter.com/userguide3/general/security.html?highlight=index%20html#hide-your-files